src/textual/widgets/_markdown.py

```diff
@@ -509,6 +514,7 @@ def __init__(
         name: str | None = None,
         id: str | None = None,
         classes: str | None = None,
+        parser_factory: Callable[[], MarkdownIt] = default_parser_factory,
```

Comment:
Can we make this None-able. When you need to construct the parser, check for None and create `MarkdownIt("gfm-like")`.

This means we could drop the `default_parser_factory` function.

Reply:
I can rewrite it this way, sure.

Sorry for prematurely requesting your re-review.